.github/workflows/tests.yml

```diff
@@ -34,7 +31,13 @@ jobs:
          version: '3.13.0'
 
     - name: Execute unit-tests
-      run: make unittest
+      run: python3 -m pytest -s -vvv tests/unit/
 
     - name: Execute integration-tests
-      run: make integrationtest
+      run: python3 -m pytest -s -vvv tests/integration/ --log-dir=/tmp/ci-logs
```

Comment:
it is more straight forward to configure the tests with flags for pytest instead of env variables for the Makefile